be/src/olap/delta_writer.cpp

```diff
@@ -172,15 +164,18 @@ OLAPStatus DeltaWriter::close(google::protobuf::RepeatedPtrField<PTabletInfo>* t
             return st;
         }
     }
-    RETURN_NOT_OK(_mem_table->close(_writer));
+    RETURN_NOT_OK(_mem_table->close(_rowset_builder.get()));
 
     OLAPStatus res = OLAP_SUCCESS;
     //add pending data to tablet
-    RETURN_NOT_OK(_tablet->add_pending_version(_req.partition_id, _req.transaction_id, nullptr));
-    for (SegmentGroup* segment_group : _segment_group_vec) {
-        RETURN_NOT_OK(_tablet->add_pending_segment_group(segment_group));
-        RETURN_NOT_OK(segment_group->load());
-    }
+    RETURN_NOT_OK(_tablet->add_pending_version(_req.partition_id, _req.txn_id, nullptr));
```

Comment:
tablet has no add_pending_version interface, you should add it into RowsetMetaManger.

Reply:
OK, I think this should be removed